redash/settings/dynamic_settings.py

```diff
@@ -10,11 +10,10 @@ def query_time_limit(is_scheduled, user_id, org_id):
     return scheduled_time_limit if is_scheduled else adhoc_time_limit
 
 
-# Provide any custom tasks you'd like to run periodically
-def custom_tasks():
-    return {
-        # 'key': {
-        #     'task': 'redash.task.example',
-        #     'schedule': timedelta(minutes=5)
-        # }
-    }
+"""You can provide any custom jobs you'd like to run periodically here. For example:
```

Comment:
Probably better to move it into `redash.scheduler` as a docstring for the `schedule` function?

Reply:
Not sure about this. `dynamic_settings.py` is sort of a template for users to follow when they want dynamic settings. We're sort of expecting people to create a copy of this module, modify it and point their `settings` to the modified module. Moving this piece of documentation to `redash.schedule` would make it hard find.

Reply:
The chances of someone finding it here or elsewhere are very low and therefore similar. When we would like people to find this (when it's stable enough), we will just add it to the Knowledge Base and document properly.

Reply:
c27ec3d8

Reply:
Unfortunately I had to revert this in 7a59113 (but with a double-dispatch approach) to allow periodic jobs to be scheduled when app is ready.